backend/estimate-service/src/main/java/org/egov/works/service/EnrichmentService.java

```diff
@@ -135,6 +132,23 @@ public void enrichUpdateEstimate(EstimateRequest request) {
         AuditDetails auditDetails = estimateServiceUtil.getAuditDetails(requestInfo.getUserInfo().getUuid(), estimate, false);
 
         estimate.setAuditDetails(auditDetails);
+
+        enrichUpdateEstimateWorkFlowForActionReject(request);
+    }
+
+    /**
+     * If the workflow action is 'REJECT' then assignee will be updated
```

Comment:
The assignee will be the user who created the estimate. It shouldn't be assigned to the role EST_CREATOR.

Reply:
done.